crates/tauri/src/webview/mod.rs

```diff
@@ -1157,6 +1158,11 @@ impl<R: Runtime> Webview<R> {
     Ok(())
   }
 
+  /// Returns all cookies for a specified URL including HTTP-only and secure cookies.
```

Comment:
Can you add a note about the scheme requirements here? imo quite important

Reply:
I've added a note. This actually seems more of a bug/runtime limitation than anything. I would imagine this is because the `tauri://` protocol behaves like the the `file://` protocol in browsers and doesn't actually expose a functional cookie store.